frontend/tablet/App.tsx

```diff
@@ -1,20 +1,18 @@
 import React from 'react'
-import {
-  StyleSheet, Text,
-} from 'react-native'
+import { NavigationContainer } from '@react-navigation/native'
+import { createStackNavigator } from '@react-navigation/stack'
+import { RoomDetailScreen } from './src/screens'
+
+const {Navigator, Screen} = createStackNavigator();
 
 const App: React.FC = () => {
   return (
-    <Text style={styles.text}>Hello World!</Text>
+    <NavigationContainer>
+        <Navigator screenOptions={{headerShown: false}}>
+            <Screen name="room_detail_screen" component={RoomDetailScreen}></Screen>
```

Comment:
room_detail_screen -> RoomDetailScreen. There are no snake case names in js. Only constants can be all caps snake case.